integration_tests/src/main/python/sort_test.py

```diff
@@ -19,7 +19,7 @@
 from marks import *
 from pyspark.sql.types import *
 import pyspark.sql.functions as f
-from spark_session import is_before_spark_310
+from spark_session import is_before_spark_311
```

Comment:
Do you want me to revert these changes? I thought to change it because 310 wasn't released

Reply:
No need to revert it. Just would have been nice to have it called out in the PR so I know that it is in there instead of scratching my head as to why there are changes in here that don't look related.